PinscapeConfigTool/html/Configure.htm

```diff
@@ -6167,15 +6249,23 @@ <h1>
         function mainBoardHoverName(pin)
         {
             // if there's an out port alias, return the jumper entry
-            var alias = outPortAlias[pin];

```

Comment:
Could you explain this change?

Reply:
I was trying something and forgot to change it back. The code should stay as
<code>var alias = outPortAlias[pin];</code>
for all three functions: mainBoardHoverName(), aioBoardHoverName(), and liteBoardHoverName()

Reply:
Great, I'll go ahead and roll it back.

Reply:
The binary download appears to be missing the four new images. Please refresh.